static/js/lib/validation/profile.js

```diff
@@ -299,5 +299,10 @@ export const validateFinancialAid = (edit: FinancialAidState): FinancialAidValid
   if (!edit.checkBox) {
     errors['checkBox'] = 'You must agree to these terms';
   }
+
+  if (edit.income && !/^\d+$/.test(edit.income)) {
```

Comment:
@aliceriot `!/^\d+$/.test(edit.income)` this give us all string which are not number, including `dot(.)` special characters.

Otherwise we have to cater those in our regex